### PR TITLE
Expose postgres DB on port 25432 instead of 5432

### PR DESCRIPTION
**Pull request by epapbak**
# Description

The exposed port is not used by any of the automation of this repository. Instead, it is just used for debugging when needed. Let's use 25432 so it does not conflict with any locally running PSQL server

## Type of change

- Refactor (refactoring code, removing useless files)

## Testing steps

Tested locally

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container
* [ ] new tests have been included in scenario list (make update-scenarios)

### CODE REVIEW

**Review by JiriPapousek — APPROVED** [review]

**Review by joselsegura — CHANGES_REQUESTED** [review]
Please, don't merge. 

I'm not sure about the port is not used by the automation. IIRC some of the scripts define a bunch of environment variables and maybe there the potr is being defined

**Comment by joselsegura** [comment]
I just reviewed and I can see that the port is used in:
- Dockerfile, as environment variable that will be used by any process running the container image
- Lot of *.toml configuration files for the programs under test
- The environment variable defined in Dockerfile is being used by Behave Python code (environment.py)

**Review by tisnik — APPROVED** [review]
LGTM

**Review by joselsegura — APPROVED** [review]
Sorry, I didn't realize that for docker-compose containers de port will remain untouched, it's just a change if you try to connect from the host.

Sorry for delaying you

**Comment by epapbak** [comment]
> I just reviewed and I can see that the port is used in:
> 
> * Dockerfile, as environment variable that will be used by any process running the container image
> * Lot of *.toml configuration files for the programs under test
> * The environment variable defined in Dockerfile is being used by Behave Python code (environment.py)

@joselsegura I think there's a misunderstanding. When using the `port` keyword and not the `expose` keyword in the docker-compose file, the format is HOST:CONTAINER, the services connect between each other using the value defined on the right, which represents the `container's port`. The other value would be used to access a container from the host machine. So this should work fine. But I think I'll do more testing, especially in the CI env, before merging, so we're more sure of this change =)

**Comment by epapbak** [comment]
tested by running the tests within docker and no local postgres running

**Pull request merged**
